frontend/src/scenes/experiments/Experiments.tsx

```diff
@@ -211,11 +221,14 @@ export function Experiments(): JSX.Element {
                     { key: ExperimentsTabs.Yours, label: 'Your experiments' },
                     { key: ExperimentsTabs.Archived, label: 'Archived experiments' },
                     { key: ExperimentsTabs.Holdouts, label: 'Holdout groups' },
+                    { key: ExperimentsTabs.SavedMetrics, label: 'Saved metrics' },
```

Comment:
should the tab show up without the flag?

Reply:
Thanks, will do!